nemo_skills/inference/model/vllm.py

```
@@ -24,6 +28,68 @@
+                        "type": "image_url",
+                        "image_url": {"url": base64_url},
+                    }
+                    for key in image_url:
+                        if key != "url":
+                            processed_item["image_url"][key] = image_url[key]
+                    item = processed_item
+                except FileNotFoundError:
+                    LOG.error(
+                        f"Image file not found: {resolved_path} "
+                        f"(original path: {url}, data_dir: {data_dir or 'not set'})"
+                    )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)